### DISCUSSION TITLE
Drop mempool transactions from bundle pricing denominator?

### DISCUSSION THREAD

**Discussion by llllvvuu** [flashbots/pm#110] [Q&A] [answered]
In Flashbots' [bundle pricing](https://docs.flashbots.net/flashbots-auction/searchers/advanced/bundle-pricing), mempool transactions are dropped from the numerator, in order to close the [bundle stuffing loophole from v0.1](https://github.com/flashbots/pm/discussions/53). However, shouldn't these transactions also be dropped from the denominator? i.e.

current: $(\Delta_coinbase + \sum_{T \in (U - M)} g_TmT) / (\sum_{T \in U} g_T)$
proposed: $(\Delta_coinbase + \sum_{T \in (U - M)} g_TmT) / (\sum_{T \in (U - M)} g_T)$

otherwise, the pricing seems punitive to larger (i.e. multi-tx) backruns and sandwiches. It also forgoes miner revenue from these otherwise profitable bundles

proof-of-concept:  https://github.com/flashbots/mev-geth/pull/128

**Comment by Ruteri** [comment, marked as answer]
It's not immediately clear to me that this will avoid the suffing. Discounting the total gas used opens the pricing algorithm to piggybacking mempool transactions on the priority of the bundle.
Say you have a bundle that pays 1 ETH, and it's the most profitable bundle for current block.
You can then submit any amount of transactions to the mempool at the base fee, and construct a bundle that will consist of the direct transfer plus all of the mempool transactions - say enough to fill the block, or at least for the miner not to consider other bundles.
This bundle would win with all other bundles, even if there are bundles paying only slightly less for much less gas.

**Reply by llllvvuu** [reply]
Ah you're right, this would be a different form of bundle stuffing (can't reduce the price of the searcher tx's, but can reduce the price of friendly mempool tx's) :(
